mdsuite/graph_modules/molecular_graph.py

```diff
@@ -84,7 +92,8 @@ def get_neighbour_list(positions: tf.Tensor, cell: list = None) -> tf.Tensor:
             r_ij_mat -= r_ij_mat.transpose(1, 0, 2)
 
         """
-        r_ij_matrix = tf.reshape(positions, (1, len(positions), 3)) - tf.reshape(positions, (len(positions), 1, 3))
+        r_ij_matrix = tf.reshape(positions, (1, len(positions), 3)) - \
+                      tf.reshape(positions, (len(positions), 1, 3))
```

Comment:
I like this change, it clearly improves readability.
I also tested it an `tf.reshape` appears to be faster than `tf.transpose` even though this looks like a case for transpose.

Reply:
Maybe we should make this a utils function, so it can be applied for the molecular graph as well as the RDF?

https://github.com/zincware/MDSuite/blob/8c18b998219ce8b6f01483e094e5fd672cd1fb60/mdsuite/calculators/radial_distribution_function.py#L398-L412

Reply:
We can discuss further